languages/java/src/main/java/bit/sdk/BitwardenClient.java

```diff
@@ -0,0 +1,72 @@
+package bit.sdk;
+
+import bit.sdk.schema.AccessTokenLoginRequest;
+import bit.sdk.schema.ClientSettings;
+import bit.sdk.schema.Command;
+import bit.sdk.schema.Converter;
+import bit.sdk.schema.ResponseForAPIKeyLoginResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.sun.jna.Native;
+import com.sun.jna.Pointer;
+import java.util.function.Function;
+
+public class BitwardenClient implements AutoCloseable {
+
+    static <T, R> Function<T, R> throwingFunctionWrapper(ThrowingFunction<T, R, Exception> throwingFunction) {
+
+        return i -> {
+            try {
+                return throwingFunction.accept(i);
+            } catch (Exception ex) {
+                throw new RuntimeException(ex);
+            }
+        };
+    }
+
+    private Pointer client;
+
+    private BitwardenLibrary library;
+
+    private CommandRunner commandRunner;
+
+    private boolean isClientOpen;
+
+    private Projects projects;
+
+    private Secrets secrets;
+
+    public BitwardenClient(ClientSettings clientSettings) throws JsonProcessingException {
+        System.setProperty("jna.library.path", System.getProperty("user.dir") + "/libs");
```

Comment:
We'll want a way to bundle all the native libs in the jar and use the appropriate binary per platform and architecture.

Reply:
I am waiting on the resolution of .NET pull request from Boris https://github.com/bitwarden/sdk/pull/193
The same workflow and artifacts for multiplatform builds should be reused for the Java wrapper as well. For now, only one combination (ARM64 macos build) is coppied from the regular cargo build target, and packed with JAR. Once the multiplatform workflow is available, all relevant artifacts should be included, and a new workflow for Maven publish should be created.